src/Illuminate/Filesystem/Filesystem.php

```diff
@@ -311,6 +312,28 @@ public function size($path)
     }
 
     /**
+     * Get the size of a given directory.
+     *
+     * @param  string  $path
+     * @throws InvalidArgumentException
```

Comment:
It's unnecessary because I did `use InvalidArgumentException` which you asked.
![screenshot_1](https://user-images.githubusercontent.com/1434668/27180936-14e67526-51de-11e7-9274-745ce68ecc12.png)